editor/src/core/shared/math-utils.ts

```diff
@@ -925,6 +925,16 @@ export function clampValue(value: number, minimum: number, maximum: number): num
   return Math.max(Math.min(value, maximum), minimum)
 }
 
+export function wrapValue(value: number, minimum: number, maximum: number): number {
+  if (value > maximum) {
+    return minimum
```

Comment:
If this is wrapping, should this be doing something like `minimum + (value - maximum)` (with a little recursion sprinkled in)?

Reply:
true, it was a very quick implementation that does not count for the overflow value. will fix :)

Reply:
https://github.com/concrete-utopia/utopia/pull/3167/commits/16e855dd3122a67f29bbf0264ee3f34072ca654e